### ISSUE TITLE
ui: add stats for internal traffic

### ISSUE THREAD

**Issue by cuongdo** [cockroachdb/cockroach#5237]
Carried over from #2721:
- [ ] Total reqs count/bytes in /bytes out
- [ ] Req Reads count/ bytes out
- [ ] Req Writes count / bytes in

**Comment by cuongdo** [comment]
removed the existing milestone, which was applied last year

@vivekmenezes please triage

**Comment by spencerkimball** [comment]
@petermattis is this really 1.0? What do we mean by internal requests? KV reqs or Raft reqs?

**Comment by petermattis** [comment]
We have metrics on `Node.Batch`. Probably some other work to do there to measure traffic. @mberhault Weren't you planning on hooking into gRPC interceptors?

**Comment by mberhault** [comment]
yup, this depends on #8093. It would be nice to have, but I'm not sure this will be done by 1.0.

**Comment by dianasaur323** [comment]
If this depends on #8093, which is still open, can we punt to 1.2? I'm also tempted to say that if we haven't done this yet, it probably isn't mission critical.

**Comment by cuongdo** [comment]
Yes, 1.2 is fine

**Comment by dianasaur323** [comment]
👍
